app/businesses/[slug]/page.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/no-unescaped-entities */
+import Image from 'next/image';
+import Link from 'next/link';
+import websiteLogo from '@/public/images/website-logo.png';
+import prisma from '@/lib/prisma';
+
+interface BusinessDetailProps {
+  params: {
+    slug: string;
+  };
+}
+
+export async function generateMetadata({
+  params: { slug },
+}: BusinessDetailProps) {
+  const business = await prisma.business.findUnique({
+    where: {
+      slug,
+    },
+    select: {
+      name: true,
+    },
+  });
+
+  return {
+    title: `Support ${business?.name}`,
+  };
+}
+
+function RelatedBusinesses({ category, relatedBusinesses }: any) {
+  return (
+    <div className="col-md-4">
+      <div className="sidebar-blk">
+        <h4>Related Listings</h4>
+        <p>
+          While you're here, be sure to check out these other{' '}
+          <span className="category">{category}</span> listings!
+        </p>
+        <ul className="related-business-list">
+          {relatedBusinesses.map((relatedBusiness: any) => (
+            <li key={relatedBusiness.id}>
+              {/* <Link href={relatedBusiness.urlName}>
+                <Image
+                  className="related-image"
+                  src={relatedBusiness.image.fluid}
+                />
+              </Link> */}
+              <div className="related-information">
+                <Link href={`/businesses/${relatedBusiness.slug}`}>
+                  <span>{relatedBusiness.name}</span>
+                </Link>
+              </div>
+            </li>
+          ))}
+        </ul>
+      </div>
+    </div>
+  );
+}
+
+export default async function BusinessDetail({
```

Comment:
We'll want to separate these out into different files. This looks like our main router here then?